packages/monorepo-tools/bin/monorepo-tools.js

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
```

Comment:
This file is missing the +x flag, either the flag should be added or this line should be removed (I’d strongly lean towards the former :slightly_smiling_face:)